app/src/org/commcare/dalvik/activities/CommCareHomeActivity.java

```diff
@@ -882,7 +882,12 @@ public void onClick(DialogInterface dialog, int i) {
                 });
                 return;
             }
-            startFormEntry(CommCareApplication._().getCurrentSessionWrapper());
+            AndroidSessionWrapper asw = CommCareApplication._().getCurrentSessionWrapper();
+            // COMMCARE-167204 by dcluna on 06/10/2015:
+            // only try to start form entry if there is an actual form behind this
+            if(asw.getSession().getForm() != null) {
+                startFormEntry(asw);
+            }
```

Comment:
We should throw an error here likely if there wasn't a form. We shouldn't be able to get here in the first place.